internal/broker/data.go

```diff
@@ -0,0 +1,58 @@
+package broker
```

Comment:
Can you rename the file to something less generic than `data`? this file seems to only deal with userinfo and errors, so maybe `authresponse.go`?